src/main/resources/META-INF/rewrite/dropwizard-to-spring-boot.yml

```diff
@@ -135,6 +146,13 @@ recipeList:
   - org.openrewrite.java.dropwizard.annotation.AddClassAnnotationIfSuperTypeExists:
       annotationToAdd: "org.springframework.transaction.annotation.Transactional"
       targetSupertypeName: "io.dropwizard.hibernate.AbstractDAO"
+  - org.openrewrite.java.dropwizard.method.ChangeSuperType:
+      targetClass: "io.dropwizard.hibernate.AbstractDAO"
+      newSuperclass: "ee.taltech.example.AbstractJpaDAO" # Custom class; from helper library
```

Comment:
I'm not seeing any reference to this one; might that be internal to what you're developing there?

Reply:
Understandable that you'd want to trim that down quite a bit, especially in house. For the recipes we offer here I think it might be best not to make any surprising code changes for folks, so I've commented out this step for now so we can get the rest of the changes in.